outputs/file/file.go

```diff
@@ -121,7 +134,18 @@ func (f *File) Init(ctx context.Context, cfg map[string]interface{}, opts ...out
 }
 
 // Write //
-func (f *File) Write(_ context.Context, rsp proto.Message, meta outputs.Meta) {
+func (f *File) Write(ctx context.Context, rsp proto.Message, meta outputs.Meta) {
+	err := f.sem.Acquire(ctx, 1)
+	if errors.Is(err, context.Canceled) {
+		return
+	}
+
+	if err != nil {
+		f.logger.Printf("failed marshaling proto msg: %v", err)
```

Comment:
This log message needs to be changed to something more meaningful, maybe just a  `failed write:%v` ?